sky/serve/controller.py

```diff
@@ -120,8 +120,12 @@ async def update_service(request: fastapi.Request):
                 logger.info(
```

Comment:
Where do we set the version in the database? Should we set the version/spec in the database in the controller, before we call into the `update_version` for replica manager and autoscaler, both for readability and correctness in concurrent setting?